app/models/green_flag/feature.rb

```diff
@@ -61,7 +63,23 @@ def fully_enabled?
   def fully_disabled?
     rules.count == 0 || rules.all? { |rule| rule.percentage == 0 }
   end
-  
+
+  def destroyable?
+    fully_enabled? || fully_disabled?
+  end
+
+  def requires_manual_deletion?
+    feature_decisions.count >= DELETION_MAX_FEATURE_DECISIONS
+  end
```

Comment:
Whoops, good catch. I had made a backup before I got repo access and apparently opened the backup file for that change.